CHANGELOG.md

```diff
@@ -8,10 +8,15 @@
 
 ## master
 
+## 1.1.3
+
+### Fixed
+- Fix creating invalid keys, if contain quote marks ([#22](https://github.com/AckeeCZ/ACKLocalization/pull/22))
```

Comment:
You're just missing `kudos @mateuszjablonski` here 🙂

Reply:
updated 👍